contrib/devtools/github-merge.py

```
@@ -51,10 +52,45 @@ def git_config_get(option, default=None):
+    Returns None — leaving urllib's default behavior untouched — when:
+      * SSL_CERT_FILE or SSL_CERT_DIR is set (respect explicit user config),
+      * the default trust store already has CA certs, or
+      * certifi is not installed.
+    '''
+    if 'SSL_CERT_FILE' in os.environ or 'SSL_CERT_DIR' in os.environ:
+        return None
+    try:
+        default_ctx = ssl.create_default_context()
+        if default_ctx.cert_store_stats().get('x509_ca', 0) > 0:
+            return None
+    except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)